src/setts/setts.service.ts

```diff
@@ -82,27 +86,46 @@ export class SettsService {
       sources: [],
     };
 
-    const [settSnapshots, settData] = await Promise.all([
-      this.getSettSnapshots(settName, SAMPLE_DAYS),
-      getSett(chain.graphUrl, settDefinition.settToken),
-    ]);
+    const record = await query({
```

Comment:
can we encapsulate this into a helper within the util file? i can see this being useful elsewhere probably.

Reply:
Discussed offline - will move to an AWS / Dynamo service in later PRs